astropy/io/misc/asdf/tags/transform/basic.py

```diff
@@ -91,15 +90,16 @@ def _to_tree_base_transform_members(cls, model, node, ctx):
             if bounds_nondefaults:
                 node['bounds'] = bounds_nondefaults
 
+        return node
+
     @classmethod
     def to_tree_transform(cls, model, ctx):
         raise NotImplementedError("Must be implemented in TransformType subclasses")
 
     @classmethod
     def to_tree(cls, model, ctx):
         node = cls.to_tree_transform(model, ctx)
-        cls._to_tree_base_transform_members(model, node, ctx)
-        return node
+        return cls._to_tree_base_transform_members(model, node, ctx)
```

Comment:
Yeah, I only changed this one because I thought it would be confusing if `_to_tree_base_transform_members` and `_from_tree_base_transform_members` worked differently, and I needed to return the latter to make the generator work.